ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -156,6 +156,12 @@
 ### Router BGP EVPN Address Family
 {%     if router_bgp.address_family_evpn.evpn_hostflap_detection is arista.avd.defined %}
 
+{%         if router_bgp.address_family_evpn.route.import_match_failure_action is arista.avd.defined('discard') %}
```

Comment:
I don't think it is on purpose, that you enclose this `if` inside the `evpn_hostflap_detection`. So move this above the `if` in line 157, and then add a new `if` around this section (including 1 leading empty line) `if router_bgp.address_family_evpn.route.import_match_failure_action is arista.avd.defined`. 
This extra `if` serves the purpose of not changing documentation for all devices where this feature is not specifically enabled or disabled.